api/v1/helpers/verbs/postUtils.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * api/v1/helpers/verbs/postUtils.js
+ */
+'use strict';
+const NOT_FOUND = -1;
+const redisModelSample = require('../../../../cache/models/samples');
+const sampleStore = require('../../../../cache/sampleStore');
+const constants = require('../../constants');
+const logAPI = require('../../../../utils/apiLog').logAPI;
+const publisher = require('../../../../realtime/redisPublisher');
+const realtimeEvents = require('../../../../realtime/constants').events;
+const authUtils = require('../authUtils');
+const u = require('./utils');
+const featureToggles = require('feature-toggles');
+
+/**
+ * @param {Object} params From swagger
+ * @param {Object} props From express
```

Comment:
"props"  is not from express. I think it is the model on which the post action is happening.